docs-web/src/main/webapp/src/locale/de.json

```
@@ -118,7 +118,9 @@
         "drop_zone": "Drag & Drop Dateien hierherziehen, um diese hochzuladen",
         "add_files": "Dateien hinzufügen",
         "file_processing_indicator": "Diese Datei wird gerade bearbeitet. Die Suche wird nicht verfügbar sein, bevor der Vorgang abgeschlossen ist.",
-        "reprocess_file": "Diese Datei erneut verarbeiten"
+        "reprocess_file": "Diese Datei erneut verarbeiten",
+		"upload_new_version": "Neue Version hochladen",
```

```suggestion
        "upload_new_version": "Neue Version hochladen",
```
Spaces for indentation.